ui/client/components/toolbars/process/buttons/UnArchiveButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import {RootState} from "../../../../reducers/index"
+import {connect} from "react-redux"
+import * as DialogMessages from "../../../../common/DialogMessages"
+import HttpService from "../../../../http/HttpService"
+import history from "../../../../history"
+import {events} from "../../../../analytics/TrackingEvents"
+import {toggleConfirmDialog} from "../../../../actions/nk/ui/toggleConfirmDialog"
+import {bindActionCreators} from "redux"
+import ToolbarButton from "../../../toolbarComponents/ToolbarButton"
+import {isArchived, getProcessId, isSubprocess} from "../../../../reducers/selectors/graph"
+import {useTranslation} from "react-i18next"
+import {ReactComponent as Icon} from "../../../../assets/img/toolbarButtons/unarchive.svg"
+import {ProcessesTabData} from "../../../../containers/Processes"
+import {SubProcessesTabData} from "../../../../containers/SubProcesses"
+
+function UnArchiveButton(props: StateProps) {
+  const {
+    processId, isArchived,
+    toggleConfirmDialog,
+  } = props
+  const redirectPath = isSubprocess ? ProcessesTabData.path : SubProcessesTabData.path
+  const {t} = useTranslation()
+
+  return (
+    <ToolbarButton
+      name={t("panels.actions.process-unarchive.button", "unarchive")}
+      icon={<Icon/>}
+      disabled={!isArchived}
+      onClick={() => isArchived && toggleConfirmDialog(
+        true,
+        DialogMessages.unArchiveProcess(processId),
+        () => HttpService.unArchiveProcess(processId).then(() => history.push(redirectPath)),
+        t("panels.actions.process-archive.yes", "Yes"),
```

Comment:
Why two lines above we have DialogMessages.unArchiveProcess, and here we inline?